src/components/Footer/index.jsx

```diff
@@ -1,25 +1,28 @@
 import React from "react";
 import "./index.scss";
 import { Container, Row, Col } from "react-bootstrap";
+import { useTranslation } from "react-i18next";
 import logo from "../../images/logoFooter.png";
 import twitterIcon from "../../images/icons/twitterIcon.svg";
 import linkedInIcon from "../../images/icons/linkedInIcon.svg";
 import instagramIcon from "../../images/icons/instagramIcon.svg";
 import facebookIcon from "../../images/icons/facebookIcon.svg";
 
 const Footer = () => {
+  const { t } = useTranslation();
+
   return (
     <footer className="mainFooter">
       <Container>
         <Row>
           <Col>
             <img src={logo} alt="Logo" className="logo" />
           </Col>
-          <Col className="footerInfo">
-            <a href="/privacy">Privacy and Cookies</a>
-            <a href="/terms">Terms and Conditions</a>
-            <p>Follow us on social media:</p>
-            <div>
+          <Col md={3} className="footerInfo">
+            {/* <a href="/privacy">Privacy and Cookies</a>
```

Comment:
we can delete it

Reply:
never commit commented out code (it's probably been at least 7 years since i've done that, so i think "never" is a safe word here).

if you need it just safe it or you can rederive it from the git history, e.g., "git diff af4t2t Footer/index.js" or something like this

Reply:
no I dont need it, it was a mistake